### PR TITLE
Initialise README template

### LINKED ISSUE #1: Create template for README
Create a template for the README which outlines the main sections of documentation we need to complete in the future for this project

### PR DESCRIPTION
**Pull request by Ashwin-MJ**
Created a template using [this](https://github.com/othneildrew/Best-README-Template). Lots more to be added as we go along, but it has the main sections in place for us to complete in the future

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,146 @@
...
+
+<!-- CONTACT -->
+## Contacts
+
+This project is being completed by a team of 5th Year Engineering students at the University of Glasgow:
+
+* [Andrew Ritchie](https://github.com/Andrew-Ritchie) 
+* [Ashwin Maliampurakal](https://github.com/Ashwin-MJ)
+* [Conor Begley](https://github.com/C-Begley)
+* [Miklas Reichmann](https://github.com/miklasr)
```
Comment (miklasr) [inline README.md]: Fix spelling: R~ei~iechmann

File: README.md [hunk 2]
```diff
@@ -1,2 +1,146 @@
...
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+1. Fork the repo
+2. Clone the repo
+   ```sh
+   git clone https://github.com/your_username_/PosturePerfection.git
```
Comment (miklasr) [inline README.md]: `https://github.com/ESE-Peasy/PosturePerfection.git`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,146 @@
...
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
```
Comment (miklasr) [inline README.md]: Replace with `TBC` or similar

File: README.md [hunk 4]
```diff
@@ -1,2 +1,146 @@
...
+
+To get PosturePerfection running by yourself you will need to follow these steps.
+
+### Prerequisites
+
+Specify which software dependencies are used and how to install them:
+
+  ```sh
+  npm install npm@latest -g
+  ```
```
Comment (miklasr) [inline README.md]: Replace with TBC or similar

**Review by miklasr — CHANGES_REQUESTED** [review]
It's a really nice template, just a few minor things I'd change.

If we add in **TBC** in places that aren't complete yet they won't be missed as easily later on. I'd prefer not leaving in incorrect installation steps like `npm install ...`

Also my name spelling

File: README.md (hunk 1 above)
Comment (Ashwin-MJ) [reply README.md]: Oh no, apologies for this!

**Review by miklasr — APPROVED** [review]
Looks good

**Pull request merged**